index.html

```diff
@@ -0,0 +1,133 @@
+                    <img src="./Images/lecture.png" class="event-logo" alt="lecture">
+                    <h6 class="sixth-header">Lecture</h6>
+                    <p class="fifth-para">Listen to the minister of power and other notable figures in the the power sector give update on the current electrical situation of the country.</p>
+                </div>
+                <div class="event-1">
+                    <img src="./Images/forum.png" class="event-logo" alt="forum">
+                    <h6 class="sixth-header">Forum</h6>
+                    <p class="fifth-para">Ask question, have discussions with the country's leading power officials here to guide you and answer all questions you might have.</p>
+                </div>
+                <div class="event-1">
+                    <img src="./Images/workshop.png" class="event-logo" alt="workshop">
+                    <h6 class="sixth-header">Workshop</h6>
+                    <p class="fifth-para">Network with top tech leaders accross the country. Display the country's future electrical system.</p>
+                </div>
+                <div class="event-1">
+                    <img src="./Images/stage.png" class="event-logo" alt="stage">
+                    <h6 class="sixth-header">Exhibition</h6>
+                    <p class="fifth-para">Exhibition of the transformations taken place in the country's power sector.</p>
+                </div>
+                <div class="event-1">
+                    <img src="./Images/febeee9e-160f-43c3-8157-cc67d2ef25bd.svg" class="award" alt="awards">
+                    <h6 class="sixth-header">Awards</h6>
+                    <p class="fifth-para">Rewarding and recognising the top players in the country's energy sector.</p>
+                </div>
+            </div>
+            <div class="whole-hr">
+            <p class="whole-program">See The Whole Program</p>
+            <div class="hr2"></div>
+            </div>
+            <button id="button-a" type="button">Join N.I.E.S 2022</button>
+        </section>
+        <section id="featured">
+            <div class="eight">
+                <h3 class="eighth-header">Featured Speakers</h3>
+                <div class="hr1"></div>
+            </div>
+            <div id="speakers">
+
+            </div>
+        </section>
+        <section id="partners">
+            <h3 class="ninth-header">Partners</h3>
+            <div class="hr1"></div>
+            <ul id="partners-list">
+            <li><img class="partner" src="./Images/dangote3.png" alt="dangote logo"></li>
+            <li><img class="partner" src="./Images/google3.png" alt="google logo"></li>
+            <li><img class="partner" src="./Images/zenith.png" alt="zenith logo"></li>
+            <li><img class="partner" src="./Images/forte-oil.png" alt="forte-oil logo"></li>
+            <li><img class="partner" src="./Images/mtn.png" alt=""></li>
+            </ul>
+        </section>
```

Comment:
- [ ] In desktop width, partners section should be invisible and the footer should be dark:

Your design  | Behance design
------------- | -------------
<img width="1025" alt="Screenshot 2022-04-13 at 17 35 37" src="https://user-images.githubusercontent.com/77038610/163192592-71e7737e-c89d-423a-a068-6c702b2f6ae1.png">  | <img width="848" alt="Screenshot 2022-04-13 at 17 35 23" src="https://user-images.githubusercontent.com/77038610/163192529-887569bf-7c6b-4e7a-a74c-a0587284cb95.png">